address/address_v1.go

```diff
@@ -70,12 +69,12 @@ func (addr *AddrV1) String() string {
 	// Group the payload into 5 bit groups.
 	grouped, err := bech32.ConvertBits(payload, 8, 5, true)
 	if err != nil {
-		log.L().Panic("Error when grouping the payload into 5 bit groups.", zap.Error(err))
+		panic("Error when grouping the payload into 5 bit groups." + err.Error())
```

Comment:
why not log.L().Panic?

Reply:
so we don't need to import uber/zap package
feel it's like killing chicken using butcher knife :)

changed to log.Panic()